x-pack/solutions/observability/plugins/observability/public/pages/rule_details/components/header_actions.tsx

```diff
@@ -48,8 +52,18 @@ export function HeaderActions({
   const [snoozeModalOpen, setSnoozeModalOpen] = useState<boolean>(false);
   const [isUntrackAlertsModalOpen, setIsUntrackAlertsModalOpen] = useState<boolean>(false);
 
+  const { euiTheme } = useEuiTheme();
+
+  const collapsedItemActionsCss = css`
+    .collapsedItemActions__deleteButton {
+      color: ${euiTheme.colors.textDanger};
+    }
+  `;
+
   const { mutateAsync: enableRule } = useEnableRule();
   const { mutateAsync: disableRule } = useDisableRule();
+  const { mutateAsync: runRule } = useRunRule();
+  const { mutateAsync: updateAPIKey } = useUpdateAPIKey();
```

Comment:
I don't think we use react query correctly here: https://tkdodo.eu/blog/mastering-mutations-in-react-query#mutate-or-mutateasync

since we don't handle the promises ourselves, we should rely on mutate.